src/blue-theme.scss

```diff
@@ -35,22 +35,17 @@ $primary: $democracy-blue;
     height: 28px;
 }
 
-[class*="toast-display"] {
-    position: absolute;
-    z-index: 1000000;
-}
-
-[class*="toast"] {
-    margin-bottom: 0 !important;
-}
-
 [class*="pageloader"] [class*="title"] {
     max-width: 100vw;
     width: fit-content;
     text-align: center;
     white-space: normal;
 }
 
+[class*="pageloader"] {
+    z-index: 99998;
+}
+
```

Comment:
This is some very suspicious code, but OK.

Reply:
The toast z-index is 99999, so setting the pageloader to 99998 makes the notifications appear above it.